tests/cypress/integration/frontend/recommendationsPage.ts

```diff
@@ -66,7 +66,7 @@ describe('Recommendations page', () => {
           cy.contains('Filters', {matchCase: false}).click();
 
           cy.contains('A year ago', {matchCase: false}).should('be.visible');
-          cy.get('input[type=checkbox][name=""]').check();
+          cy.contains('All time', {matchCase: false}).click();
```

Comment:
For some reason clicking on this input element would fail because the element was partially hidden :shrug: